src/zope/configuration/tests/test_config.py

```diff
@@ -93,7 +93,7 @@ def test_resolve_bad_dotted_import(self):
         # Import error caused by a totally wrong dotted name.
         from zope.configuration.exceptions import ConfigurationError
         c = self._makeOne()
-        exc = self.assertRaises(ConfigurationError, 
+        exc = self.assertRaises(ConfigurationError,
                           c.resolve, 'zope.configuration.nosuch.noreally')
```

Comment:
I find the context manager spelling of `assertRaises` much clearer, which works now that we have dropped Python 2.6 support:

```python
with self.assertRaises(ConfigurationError):
    c.resolve('zopeconfiguration.nosuch.noreally')
```